ambry-api/src/main/java/com.github.ambry/rest/RestUtils.java

```diff
@@ -416,4 +424,32 @@ public static String getOperationOrBlobIdFromUri(RestRequest restRequest, RestUt
     }
     return subResource;
   }
+
+  /**
+   * Fetch time in ms for the {@code dateString} passed in, since epoch
+   * @param dataString the String representation of the date that needs to be parsed
+   * @return Time in ms since epoch. Note http time is kept in Seconds so last three digits will be
+   *         000. Returns null in case of parse exception
```

Comment:
"Returns null if the {@code dateString} is not in the expected format or could not be parsed"